src/test/kotlin/no/skatteetaten/aurora/boober/feature/DeploymentConfigFeatureTest.kt

```diff
@@ -29,6 +29,32 @@ class DeploymentConfigFeatureTest : AbstractFeatureTest() {
         }.isSuccess()
     }
 
+    @Test
+    fun `should not allowed latest for releaseTo`() {
+        assertThat {
+            createAuroraConfigFieldHandlers(
+                """{ 
+                "groupId": "no.skatteetaten.aurora",
+                "version" : "1",
+                "releaseTo": "latest"
+               }"""
+            )
+        }.singleApplicationError("Disallowed")
+    }
+
+    @Test
+    fun `should not allow semantic version for releaseTo`() {
+        assertThat {
+            createAuroraConfigFieldHandlers(
+                """{ 
+                "groupId": "no.skatteetaten.aurora",
+                "version" : "1",
+                "releaseTo": "1.2.3"
```

Comment:
ref. kommentar om verdi 123, kan du legge inn en test for 123 verdi?

Reply:
fikset